site/e2e/api.ts

```diff
@@ -27,17 +27,19 @@ export const getCurrentOrgId = async (): Promise<string> => {
   return currentOrgId;
 };
 
-export const createUser = async (orgId: string) => {
-  const name = randomName();
-  const user = await API.createUser({
-    email: `${name}@coder.com`,
-    username: name,
-    password: "s3cure&password!",
+export const createUser = async (
+  orgId: string,
+  username: string = randomName(),
+  password: string = "s3cure&password!",
+) => {
```

Comment:
nit: instead of having the user params passed as an argument what about creating a param type that could merge/override the default values?